docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -216,101 +151,111 @@ config from cloud.resource where cloud.type = 'gcp' AND api.name = 'gcloud-armor
 |*Service*
 |*API Details*
 
-|*AWS Batch*
-//RLP-122581
+|*AWS Glue*
+
+tt:[*24.3.2*]
+
+//RLP-125110
+
 |*aws-batch-job-definition*
 
 Additional permission required:
 
-* `batch:DescribeJobDefinitions`
-
-The Security Audit role includes the permission.
+* `aws-glue-schema`
+* `glue:ListSchemas`
 
-|*AWS CodeBuild*
-//RLP-118748
+You must manually add the above permission to the CFT template to enable it, if not present in the Security Audit role.
```

Comment:
Rephrase this sentence to:
You must manually add the above permissions to the CFT template to enable them.